docs/extra/PubisherClientSnippets.md

```diff
@@ -0,0 +1,30 @@
+---
```

Comment:
These snippets are very useful. My one tweak would be to add a blank line after creating the client to visual separate that code from the code that you’re actually showing off.

On Apr 15, 2016, at 6:30 AM, Chris Bacon <notifications@github.com<mailto:notifications@github.com>> wrote:


In docs/extra/PubisherClientSnippets.md<https://github.com/GoogleCloudPlatform/gcloud-dotnet/pull/112#discussion_r59873708>:

> @@ -0,0 +1,30 @@
> +---


There's a typo in the filename ;)

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly or view it on GitHub<https://github.com/GoogleCloudPlatform/gcloud-dotnet/pull/112/files/f0ace4fc200c944212bdc2d1cbd0cd209ffc934e#r59873708>